cpe_workspaceone/resources/cpe_workspaceone.rb

```diff
@@ -43,22 +50,53 @@ def uninstall?
 
   def enforce_mdm_profiles
     return unless node['cpe_workspaceone']['mdm_profiles']['enforce']
+
     macos_enforce_mdm_profiles if node.macos?
   end
 
+  def set_cli_config(flag, val)
```

Comment:
since this isn't needed for `hubcli` to work out of the box, I'm inclined to think we should be less opinionated here. AKA, this should have a guard with an attribute of `manage_cli` set to false and only trigger these code blocks if the admin actually wants to manage this portion.

Reply:
I've added this gating, have a gander.